src/pages/pt-br/guides/deploy/render.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Faça o deploy do seu site Astro no Render
+description: Como fazer o deploy do seu site Astro usando Render.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+Você pode fazer deploy do seu projeto Astro no [Render](https://render.com/), um serviço para construir sites com certificados TLS gratuitos, uma CDN global, proteção contra DDoS, redes privadas e deploys automáticos do Git.
+
+## Como fazer o deploy
+
+1. Crie uma [conta no render.com](https://dashboard.render.com/) e faça login
+2. Clique no botão **New +** em seu dashboard e selecione **Static Site**
+3. Conecte seu repositório do [GitHub](https://github.com/) ou [GitLab](https://about.gitlab.com/) ou insira alternativamente a URL pública de um repositório público
```

Comment:
Para deixar esse item mais fácil de ler pensei nessa pequena mudança, o que você acha?
```suggestion
3. Conecte seu repositório do [GitHub](https://github.com/) ou [GitLab](https://about.gitlab.com/) ou então insira a URL pública de um repositório público
```

Reply:
Thanks, I'm glad to help! 😄